matsim/scenariogen/data/__init__.py

```diff
@@ -189,6 +191,7 @@ class TripMode(AutoNameLowerStrEnum):
     PT = auto()
     MOTORCYCLE = auto()
     OTHER = auto()
+    COLECTIVO = auto()
```

Comment:
Please use the english name. Does it not belong to PT ?
It is very relevant in your scenario? Maybe it can be subsumed under other. 
The modes are very high level in this enum. Maybe a second enum with more detailed modes is useful. SrV and Mid also have more fine-grained differentiation if needed.

Reply:
Does it run on a schedule like PT or is it similar to taxi / drt ?  Is there a way to translate it to English ?

Reply:
There are routes and (rough) schedules like in "normal" PT, but the problem is that this data is not public. So all we really can do is guessing. RIght now, I am planning to implement this mode as a teleported mode. Just adding it to the usual pt will not serve as colectivos are heavily used in areas, where no metro line / metrobus etc. is operating.

Reply:
the english translation would be something like "minibus"

Reply:
I think the german name would be Sammeltaxi and the official translation SHARE_TAXI. There are actually many countries with a similar concept.

Reply:
You are right about the german name, but in this case the vehicles are rather buses than "taxis", so I would like to go with minibus